src/components/AmountInput/AmountInput.vue

```diff
@@ -9,13 +9,28 @@
         class="inputs-container"
     >
         <ErrorTooltip :errors="errors">
-            <input v-model="relativeValue" class="input-style input-size" type="text" />
+            <input v-model="relativeValue" class="input-style amount-input-size" type="text" />
+            <div v-if="!isOffline">
+                <input v-model="totalAvailableAmount" class="input-style total-amount-input-size" :disabled="true" />
+                <p v-if="isAmountGreaterThanBalance" style="color: red;">{{ $t('not_enough_balance') }}</p>
+            </div>
         </ErrorTooltip>
     </ValidationProvider>
 </template>
 
 <script lang="ts">
 import { AmountInputTs } from './AmountInputTs';
-
 export default class AmountInput extends AmountInputTs {}
 </script>
+<style lang="less" scoped>
+.amount-input-size {
+    width: 60%;
+    height: 100%;
+}
+.total-amount-input-size {
+    width: 40%;
+    height: 100%;
+    color: #515a6e;
+    padding-left: 0.3rem;
+}
```

Comment:
Should we move the styles to `AmountInput.less` to preserve consistency across components and use as below?
```
@import './AmountInput.less';
```